primeqa/mrc/README.md

```diff
@@ -232,6 +232,22 @@ R@P=0.9: 10.16% (actual p=90.00%, score threshold=7.425)
 ```
  #### [MRQA](https://huggingface.co/datasets/mrqa)
 
+ The dataset is a collection of 18 existing QA dataset (carefully selected subset of them) and converted to the same format (SQuAD like format)
+
+ For the MRQA dataset use the following additional command line arguments:
+ ```shell
+       --dataset_name mrqa \
+       --dataset_config_name plain_text \
+       --preprocessor primeqa.mrc.processors.preprocessors.mrqa.MRQAPreprocessor 
```

Comment:
I think we also need to add postprocessor and eval_mtrics here to use squad format.

Reply:
Seems that the subset name for TriviaQA  is TriviaQA-web.

Reply:
fixed